src/App/Router.php

```diff
@@ -91,14 +108,21 @@ class Router
 	 * @param string              $baseRoutesFilepath The path to a base routes file to leverage cache, can be empty
 	 * @param L10n                $l10n
 	 * @param ICanCache           $cache
+	 * @param ICanLock            $lock
```

Comment:
This parameter doesn't exist.

Reply:
of course it does o_O see https://github.com/friendica/friendica/pull/11005/files/a7a05633d8226edb76eb9a9f0bce184db82bd4a3#diff-7f7aabf2d5c355cac62f512a0d5387e00dfcb00b27e6dcb39b3b90ff028d05dfR122